.github/workflows/analysis-of-endpoint-connections.yml

```diff
@@ -0,0 +1,26 @@
+name: Analysis-of-Endpoint-Connections
+
+on:
+  pull_request:
+    paths:
+      - 'src/main/java/**'
+      - 'src/main/webapp/**'
+
+jobs:
+  show-modified-files:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Optional: You could consider looking into a solution for this that is fast.

Reply:
Will do in a followup